frontend/pages/medical-recovery-centre/index.js

```diff
@@ -0,0 +1,111 @@
+import React from 'react'
+
+const MedicalRecoveryCentre = () => {
+  return (
+    <>
+      {style}
+      <div className="container">
+        <h1>Medical Recovery Centre</h1>
+        <TheCottage />
+        <TierneyHouse />
+        <TBAPerth />
+      </div>
+    </>
+  )
+}
+
+const TheCottage = () => {
+  return (
+    <>
+      <h2>The Cottage</h2>
+      <blockquote>
+        The Cottage is a supportive, home like environment where Complex Care
+        Services and Hospital in the Home (HITH) services are provided to people
+        who are homeless or at risk of homelessness.
+        <br />
+        <br />
+        The aim is to provide holistic, recuperative care to clients with a
+        nursing need, as an alternative to staying in hospital.
+        <br />
+        <br />
+        The Cottage focuses on building rapport and trust between clients and
+        staff, enabling staff to establish a safe environment and platform from
+        which they can address clients’ health issues.
```

Comment:
In my own opinion, it is more appropriate to use a `<p>` tag rather than using `<br>` as p represents paragraph

Reply:
You're absolutely right. And it looks cleaner with `<p>` as well!